server/models.py

```diff
@@ -1,10 +1,66 @@
-from app import db
+from flask_sqlalchemy import SQLAlchemy
+from flask_marshmallow import Marshmallow
+from marshmallow import Schema, fields
+
 from sqlalchemy.dialects.postgresql import JSON
 
+from app import db
+
+db = SQLAlchemy()
+ma = Marshmallow()
+
 class User(db.Model):
     __tablename__ = 'users'
 
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(80), nullable=False)
     email = db.Column(db.String(80), nullable=False, unique=True)
     password = db.Column(db.String(80), nullable=False, unique=True)
```

Comment:
```python
password = db.Column(db.String(80), nullable=False, unique=True)
```
This column type will be changing from String to BYTEA

Reply:
Because this will relate to your feature branch, I think it's better to leave it be and get it changed on your PR related to your sign ups. Will rebase from dev once you have merged the PR.

Reply:
Great! I was going to ask for this PR to wait until the signup backend PR gets merged in. I've filed that PR (#29 ) just now

Reply:
One other thought - this PR currently makes changes to `models.py` but doesn't include any changes to migrations. Since generating migrations appears to be buggy, I think generated, validated migrations should be included in PRs that change the models.

Reply:
Oh yes I haven't done that too. Will look up on how to do it as a git hook so this would be done automatically.

Reply:
Due to having to restart the migrations from #29, I'm holding the migrations for this PR until that is merged to `dev`. I'll rebase and migrate the db.